src/customer/strategies/amazon-maxo/amazon-maxo-customer-initialize-options.ts

```diff
@@ -0,0 +1,8 @@
+export default interface AmazonMaxoCustomerInitializeOptions {
+    /**
+     * This container is used to set an event listener, provide an element ID if you want
+     * users to be able to launch the AmazonMaxo modal by clicking on a button.
+     * It should be an HTML element.
+     */
+    container: string;
```

Comment:
Fixed, please, take a look again ♻️

Reply:
This will be kept as `container` because this value is set from checkout-js and it's common to all the payment methods with `*-customer-initalize-options` type